doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -160,6 +165,52 @@ To use the sensor trigger, complete the following steps:
 .. note::
     |only_configured_module_note|
 
+Enabling passive power management
+=================================
+
+The |sensor_manager| can react to ``power_down_event`` and ``wake_up_event``.
+This functionality is called passive power management.
+
+To use the passive power management in |sensor_manager|, complete the following steps:
+
+1. Enable :kconfig:`CONFIG_CAF_SENSOR_MANAGER_PM` option.
+   If a sensor of your choice supports trigger functionality,its enabled and configured by you, there is nothing more to do.
```

Comment:
Passive power management should have nothing common with triggers - it is part of active power management.
It would wake up the sensor only. Please note that in this configuration sensor cannot wake up the whole system.

Reply:
Passive power management (reacting to `wake_up` and `power_down` events) is different for sensor that has trigger configured and for the one that does not have it, so i assume i must explicitly write it. If the sensor has trigger we do not need `suspend_pm_state`, its ignored. If there is no trigger configuration `suspend_pm_state` should be explicitly specified by the user.